.mvn/jvm.config

```diff
@@ -1 +1,7 @@
 -XX:+IgnoreUnrecognizedVMOptions
+--add-exports=java.security.jgss/sun.security.krb5=ALL-UNNAMED
```

Comment:
Can't we add a comment here as well about the reasons for the selection here?

Reply:
I mean what was the reason for this specific selection of packages. :thinking:

Reply:
krb5 is for some reason required for the yarn kerberos tests.

The other stuff is for various plugins.

Reply:
as mentioned elsewhere we can't add a comment in this file. I will double-check whether this one specifically is still required.